data/lib/api/match/match_model.dart

```diff
@@ -103,6 +119,20 @@ extension DataMatchModel on MatchModel {
       );
     }
   }
+
+  bool get isRevisedTargetApplicable {
+    final secondInningTeam = toss_decision == TossDecision.bat
+        ? teams.firstWhere((element) => element.team.id != toss_winner_id)
+        : teams.firstWhere((element) => element.team.id == toss_winner_id);
```

Comment:
Can we do this instead, since firstWhere is being repeated here?
```suggestion
     final secondInningTeam = teams.firstWhere(
      (element) => toss_decision == TossDecision.bat
          ? element.team.id != toss_winner_id
          : element.team.id == toss_winner_id,
    );
```